src/sdc11073/xml_types/dataconverters.py

```diff
@@ -254,13 +257,13 @@ def check_valid(py_value):
 
 class DurationConverter(NullConverter):
     @staticmethod
-    def to_py(xml_value):
+    def to_py(xml_value)-> int | float | None:
```

Comment:
why not return type `x_struct.DurationType`

Reply:
because it only returns int or float, not Decimal.